app/jobs/import_positions_job.rb

```diff
@@ -0,0 +1,52 @@
+class ImportPositionsJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    response = icims_search(type: 'jobs', body: '{"filters":[{"name":"job.jobtitle","value":["successlink"],"operator":"="}]}')
+    jobs = response['searchResults'].pluck('id') - Position.all.pluck(:icims_id)
+    jobs.each do |job_id|
+      job = icims_get(object: 'jobs', id: job_id)
+      job_address = get_address_from_icims(job['joblocation']['address'])
```

Comment:
Do you think using `#try` would be worth it here? Only to protect against ICIMSness.